components/style/properties/gecko.mako.rs

```diff
@@ -598,7 +598,7 @@ def set_gecko_property(ffi_name, expr):
     // set on mContextFlags, and the length field is set to the initial value.
 
     pub fn set_${ident}(&mut self, v: longhands::${ident}::computed_value::T) {
-        use values::generics::svg::SVGLength;
+        use values::generics::svg::{SVGLength, SvgLengthOrPercentageOrNumber};
```

Comment:
@nox Why did you suggest using `Svg` ? That seems inconsistent with all the other `SVG...` types?